### ISSUE TITLE
Your repository doesn't have any source code!

### ISSUE THREAD

**Issue by msdedwards** [JackHermes/Fun_Facts#1]
Here are some notes for setting up git: https://help.github.com/articles/set-up-git/#platform-windows
This answer is a good resource for adding code into a repository, but it uses the command line:http://stackoverflow.com/questions/4658606/import-existing-source-code-to-github

**Comment by msdedwards** [comment]
Here are some nice videos:https://www.youtube.com/watch?v=FyfwLX4HAxM&list=PLg7s6cbtAD15G8lNyoaYDuKZSKyJrgwB-

**Comment by JackHermes** [comment]
Yeah, it's rather boring and empty right now. I got stuck on trying to
figure out how to add a readme. These look like they'll help lots, thanks
Matt!

In other tech project news, last week I started playing around with a
raspberry pi I got for Christmas and also the beagleboard xm Tanner
commandeered from Mac (remember that thing on the phone booth wall we tried
to get working once? Yeah!)
